src/main/java/org/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -158,13 +158,33 @@ public boolean isConnectedToDocument() {
         return xCurrentComponent != null;
     }
 
+    public static XTextDocument selectComponent(List<XTextDocument> list)
```

Comment:
Seems like I missed this Dialog in the conversion.
Could you please convert this as well?
should be a proper method in DialogService, showChoiceDialog or similar

Reply:
Maybe as an new PR. The OO integration is not really the subject of this one.